docs/tool_migration.md

```diff
@@ -71,5 +71,12 @@ Displays information about Java stack traces and thread information for processe
 
 For more information, see [`jstack`](tool_jstack.md).
 
+#### Java statistics monitoring (`jstat`) tool
+
+Displays information about Java statistics for processes. The main differences from the HotSpot `jstat` tool are as follows:
```

Comment:
Only one difference(?) so make singular:  "The main difference from the HotSpot `jstat` tool is as follows:..."
(Or just remove the (one) bullet point and merge into the sentence.)

Reply:
Currently this tool is really a start point for migration purpose, extra features can be added as per user requests.
Updated the sentence.